ci/containers/Containerfile

```diff
@@ -0,0 +1,11 @@
+FROM docker://nvidia/cuda:12.8.1-cudnn-devel-ubuntu24.04
```

Comment:
Why this image instead of just `ubuntu-24.04`? pip will install all the cuda dependencies from PyPI for us anyway

Reply:
ok, so it looks like the linux-aarch64 wheels do not have the same dependencies on the `nvidia-xxx` packages as the linux-x86_64 wheel, so we need to provide nvidia tools in the base container.